packages/ui/src/components/va-data-table/VaDataTable.demo.vue

```diff
@@ -494,9 +494,10 @@
 
 <script lang="ts">
 import { defineComponent } from 'vue'
-import cloneDeep from 'lodash/cloneDeep'
-import shuffle from 'lodash/shuffle'
+
 import VaDataTable from './'
+import cloneDeep from 'lodash-es/cloneDeep'
```

Comment:
Should fix all of our problems.

```suggestion
import cloneDeep from 'lodash/cloneDeep.js'
```